### PR TITLE
Feature/winner charge

### LINKED ISSUE #144: IN: Winner/Charge Customer

### PR DESCRIPTION
**Pull request by brian-glovebox**
### What this PR does (required):
- Can not create a contest until a card is on file 
- contest owner is charged amount of contest when the contest ends. 

### Screenshots / Videos (required):
- Post at least one screenshot to show the feature in action

### Any information needed to test this feature (required):
- create contest, submit a contest image and when contest ends choose winner, charge will happen to contest owner. 

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: server/controllers/contest.js [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+exports.createContest = asyncHandler(async (req, res) => {
+  try {
+
+    const findUser = await User.findOne({
+      _id: req.user.id
+    });
+
+    const findStripe = await stripe.customers.retrieve(findUser.stripeId);
+
+    if (findStripe.invoice_settings.default_payment_method === null) {
```
Comment (rajivtitus) [inline server/controllers/contest.js:18]: 👍🏽

File: server/utils/contestHelper.js [hunk 2]
```diff
@@ -53,15 +54,40 @@ exports.scheduleContestEnd = asyncHandler(async (contest) => {
   }
 })
 
+const stripePay = asyncHandler(async (contestOwner, prizeAmount, winnerId) => {
+  const findContestUser = await User.findOne({_id: contestOwner})
+
+  console.log('findContestUser', findContestUser)
```
Comment (rajivtitus) [inline server/utils/contestHelper.js]: Please remove and fix indentation/spacing :)

**Review by rajivtitus — APPROVED** [review]
LGTM! 👍🏽

File: server/utils/contestHelper.js (hunk 2 above)
Comment (brian-glovebox) [reply server/utils/contestHelper.js]: removed and indented

**Pull request merged**